src-electron/generator/generation-engine.js

```diff
@@ -752,7 +764,9 @@ async function generateSingleTemplate(
     disableDeprecationWarnings: false,
   }
 ) {
-  env.logInfo(`Start generating from template: ${genTemplateJsonPackage?.path}`)
+  let genStart = process.hrtime.bigint()
+  //console.log(`Start generating from template: ${singleTemplatePkg?.path}`)
```

Comment:
Should we keep the console.log? I think it is generally nice to see this during all our generations instead of seeing it in some hidden form unless logInfo does console.log.

Reply:
Console.log prints out the totals. I think it would be too much clutter if we print out these individual templates.